build/entrypoint.sh

```diff
@@ -4,12 +4,19 @@ set -e
 
 : ${GCLOUD_REGISTRY:=gcr.io}
 : ${IMAGE:=$GITHUB_REPOSITORY}
+: ${ARGS:=} # Default: empty build args
 : ${TAG:=$GITHUB_SHA}
 : ${DEFAULT_BRANCH_TAG:=true}
+: ${LATEST:=false}
 
-docker build -t $IMAGE:latest -t $IMAGE:$TAG .
+docker build $ARGS -t $IMAGE:$TAG .
 docker tag $IMAGE:$TAG $GCLOUD_REGISTRY/$IMAGE:$TAG
-docker tag $IMAGE:latest $GCLOUD_REGISTRY/$IMAGE:latest
+
+if [ $LATEST = true ]; then
+  docker build $ARGS -t $IMAGE:latest .
```

Comment:
this means building the same image twice... Did you mean to simply tag it ? 
```` bash
docker tag $IMAGE:$TAG $IMAGE:latest
docker tag $IMAGE:$TAG $GCLOUD_REGISTRY/$IMAGE:latest
```

Reply:
My bad. Yeah. I meant only tagging it as latest.